### PR TITLE
fix(docs): ecosystem section in docs sidebar

### PR DESCRIPTION
**Pull request by amberleyromo**
screen shot 2018-12-07 at 10 34 35 am

- better consistency with tutorial sidebar (intro landing page under "introduction")
- added ecosystem section which links out to starter library, plugin library, and awesome gatsby (external) resources.

### CODE REVIEW

File: www/src/pages/docs/index.js [hunk 1]
```diff
@@ -49,6 +49,10 @@ class IndexRoute extends React.Component {
                   around building with Gatsby, like sourcing data, deployment,
                   and more.
                 </li>
+                <li>
+                  <strong>Ecosystem</strong>: Check out libraries for Gatsby
```
Comment (DSchau) [inline www/src/pages/docs/index.js]: Not really new here, but is there a reason these aren't linked?

Comment (DSchau) [reply www/src/pages/docs/index.js]: I suppose because they're linked in the sidebar? But I'm not sure I see a big issue with doubling up here!

Comment (amberleyromo) [reply www/src/pages/docs/index.js]: It's because very few of these section headers are linked to actual pages (so there's nothing to link to).

Comment (amberleyromo) [reply www/src/pages/docs/index.js]: The point of including this list in the docs introduction landing page is to introduce the section areas more explicitly (people have a better idea of what they're getting).

Comment (DSchau) [reply www/src/pages/docs/index.js]: Got it; that makes sense. Thanks!

**Pull request merged**